doc/whats_new/v1.2.rst

```diff
@@ -0,0 +1,159 @@
+.. include:: _contributors.rst
+
+.. currentmodule:: sklearn
+
+.. _changes_1_2:
+
+Version 1.2.0
```

Comment:
I do not think `1.2.rst` should be in here?

Reply:
Indeed this is my first point in the todo list :)
I first try to make pass the tests :)